modules/web-server-express/package.json

```diff
@@ -16,10 +16,8 @@
   "license": "ISC",
   "dependencies": {
     "@types/cookie-parser": "^1.4.2",
-    "@types/node-fetch": "^2.5.10",
     "cookie-parser": "^1.4.5",
-    "express": "^4.17.1",
-    "node-fetch": "^2.6.1"
+    "express": "^4.17.1"
```

Comment:
`npm i` should be run in `modules/web-server-express` and the package-lock.json file committed with these changed.